src/Resources/skeleton/doctrine/broadcast_twig_template.tpl.php

```diff
@@ -0,0 +1,28 @@
+{#
+Available variables:
+* id: the normalized id of the entity
+* entity: the entity object
+* action: the current action
+* options: the options passed to the Broadcast attribute
```

Comment:
Maybe include a link to the docs? https://github.com/symfony/ux/tree/main/src/Turbo#broadcast-doctrine-entities-update

Because the user will still need to add a `{{ turbo_stream_listen(book) }}` somewhere in a template, if I understand things correctly.